spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Tracer.java

```diff
@@ -72,25 +82,33 @@
 	/**
 	 * Start a new span if the sampler allows it or if we are already tracing in this
 	 * thread. A sampler can be used to limit the number of traces created.
-	 *  @param name the name of the span
+	 *
+	 * @param name the name of the span
 	 * @param sampler a sampler to decide whether to create the span or not
 	 */
 	Span startTrace(String name, Sampler sampler);
 
 	/**
-	 * Pick up an existing span from another thread.
+	 * Pick up an existing span from another thread. Continuing a span signifies that
+	 * all changes applied to the returned span from this method will be also
+	 * applied to the continued span (the one passed as the argument).
 	 */
 	Span continueSpan(Span span);
 
 	/**
-	 * Adds a tag to the current span if tracing is currently on.
+	 * <p>Adds a tag to the current span if tracing is currently on.</p>
+	 *
+	 * <p>
+	 *     Every span may also have zero or more key/value Tags, which do not have
```

Comment:
Things like this are better said with examples, particularly, this should reference trace keys